src/app/component/shared/items/item/item.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewEncapsulation
+} from "@angular/core";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { Href } from "src/app/interfaces/menusInterfaces";
+import { StrongRoute } from "src/app/interfaces/strongRoute";
+
+@Component({
+  selector: "app-items-item",
+  template: `
+    <li class="list-group-item clearfix">
```

Comment:
Remove this `<li>` and extract it to the `<app-items>` component and any dependencies so that bootstrap styling will work.